frontend/integration-tests/tests/dashboards/cluster-dashboard.scenario.ts

```diff
@@ -30,14 +30,12 @@ describe('Cluster Dashboard', () => {
         'Service Level Agreement (SLA)',
         'Update channel',
       ];
-      await browser.wait(until.presenceOf($('[data-test-id="sla-text"]')), 7000);
       const items = clusterDashboardView.detailsCardList.$$('dt');
       const values = clusterDashboardView.detailsCardList.$$('dd');
       expect(items.count()).toBe(expectedItems.length);
       expect(values.count()).toBe(expectedItems.length);
       expectedItems.forEach(async (label: string, i: number) => {
         expect(items.get(i).getText()).toBe(label);
-        await browser.wait(waitForNone(dashboardView.loaders));
```

Comment:
Could this cause the details card items to be checked before the content loads? (I'd expect in practice the SLA text will load last, so maybe not an issue.)

Wish we had Cypress retries here :/

Reply:
It could which is why I'd previously added this to catch that issue, but assume the SLA check first will cover these, so I removed it to avoid another potential failure.